src/views/product-root-docs-path-landing/components/hero/hero.module.css

```diff
@@ -27,3 +25,8 @@
 .pageSubtitle {
 	color: var(--token-color-foreground-primary);
 }
+
+.iconCardGridLink {
+	margin-top: 32px;
```

Comment:
Moved `margin-bottom` off of `.iconTextAndWrapper` and onto `.iconCardGridLink`. This way, when `.iconCardGridLink` isn't present, we don't end up creating extra space.